.eslintrc

```diff
@@ -11,19 +11,15 @@
   ],
   "rules": {
     "no-console": 1, // Means warning
-    "prettier/prettier": 2 // Means error
+    "prettier/prettier": 1 // Means warning
```

Comment:
Would def like this to remain error. We can use per-line or per-file overrides for where we disagree.